obs-packaging/scripts/pkglib.sh

```diff
@@ -3,7 +3,8 @@
 # This is a helper library for the setup scripts of each package
 # in this repository.
 
-source_dir_pkg_lib=$(dirname "${BASH_SOURCE[ ${#BASH_SOURCE[@]} - 1 ]}")
+source_dir_pkg_lib=$(dirname "${BASH_SOURCE[0]}")
+source_dir_pkg_lib=$(realpath "${source_dir_pkg_lib}")
```

Comment:
Heh, somewhere over in the tests/metrics scripts I think I have one 'recommended method' - I did try to research what was the 'right' way to do this at one point. This looks sane though I think...

I'd suggest we research, choose, and unify - but, we have bigger fish to fry right now. For ref, over in the metrics lib
https://github.com/kata-containers/tests/blob/master/metrics/lib/common.bash#L7

there is:

```bash
THIS_FILE=$(readlink -f ${BASH_SOURCE[0]})
LIB_DIR=${THIS_FILE%/*}
source ${LIB_DIR}/../../lib/common.bash
```

Reply:
I tought about `readlink` too, but that's what you get in `man readlink`:
```
Note realpath(1) is the preferred command to use for canonicalization functionality.
```